R/object_usage_linter.R

```diff
@@ -3,6 +3,9 @@
 #' \code{\link[base]{eval}} on the code, so do not use with untrusted code.
 #' @export
 object_usage_linter <- function() {
+  if (!requireNamespace("codetools", quietly = TRUE)) {
+    stop("'object_usage_linter' depends on the codetools package to work, but it's not installed.")
```

Comment:
That's a default linter IINM, so voting for `codetools` to stay `Imports`ed.

Reply:
ditto for `cylocomp` then